app/src/main/res/values/styles.xml

```diff
@@ -31,9 +31,7 @@
     </style>
 
     <style name="AppTheme.Account.AppBarLayout" parent="ThemeOverlay.AppCompat">
-        <item name="android:textColorPrimary">@color/text_color_primary_dark</item>
-        <item name="android:textColorSecondary">@color/text_color_secondary_dark</item>
-        <item name="android:textColorTertiary">@color/text_color_tertiary_dark</item>
+        <item name="android:colorControlNormal">@color/white</item>
```

Comment:
White is okay for both themes? I'm confused tbh

Reply:
Yes this only affects the ViewMediaActivity hat is completely black regardless of theme.

Reply:
Okay, maybe we should document that so that we don't have surprises later

Reply:
true, lets rename the theme to make clear where its used